src/DynamicObj/DynamicObj.fs

```diff
@@ -27,15 +27,20 @@ type DynamicObj() =
     member this.GetValue (name) =
         this.TryGetValue(name).Value
 
-    /// Gets property value
+    /// Gets typed property value if type matches, otherwise None.
+    ///
+    /// WARNING: This Method is not supported in Fable transpiled code. Use static method tryGetTypedValue instead.
     member this.TryGetTypedValue<'a> name = 
+        #if !FABLE_COMPILER
         match (this.TryGetValue name) with
         | None -> None
-        | Some o -> 
+        | Some o ->      
             match o with
-            | :? 'a -> o :?> 'a |> Some
+            | :? 'a as o -> o |> Some
             | _ -> None
-
+        #else
+        failwith "Method TryGetTypedValue is not supported in Fable transpiled code. Use static method tryGetTypedValue instead."
```

Comment:
> Use static method tryGetTypedValue instead

the method does only exist on the `DynObj` module though, so should it not be something like 

> Method TryGetTypedValue is not supported in Fable transpiled code. Use DynObj.tryGetTypedValue instead.

?

Reply:
yes!